makecrx.sh

```diff
@@ -98,7 +98,7 @@ trap 'rm -f "$pub" "$sig" "$zip"' EXIT
 # zip up the crx dir
 cwd=$(pwd -P)
 (cd "$dir" && ../../utils/create_xpi.py -n "$cwd/$zip" -x "../../.build_exclusions" .)
-echo >&2 "Unsigned package has sha1sum: `sha1sum "$cwd/$zip"`"
+echo >&2 "Unsigned package has sha1sum: $(openssl dgst -sha1 "$cwd/$zip" | awk '{print $2}')"
```

Comment:
You use `openssl` below to generate the signature so I figured it was reasonably safe to reuse.